frontend_tests/node_tests/unread.js

```diff
@@ -8,7 +8,8 @@
 // dependencies (except _).
 
 add_dependencies({
-    muting: 'js/muting.js'
+    muting: 'js/muting.js',
+    unread: 'js/unread.js'
```

Comment:
I think this isn't needed since the unread module itself is required below, right?